### DISCUSSION TITLE
Welcome to egon.io Discussions!

### DISCUSSION THREAD

**Discussion by hofstef** [WPS/domain-story-modeler#180] [Announcements]
##  Welcome!
We’re using Discussions as a place to exchange ideas that at not feature requests/bug reports yet:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.

Have fun!
Stefan

**Comment by big-fudge** [comment]
I just had enough of the quirky draw.io for Domain story telling. Luckily I found this. 
I just want to point out a few minor flaws:
- opening the HTML export gives me the title and nothing on the next slide. Any known issues with that?
- The actor/item titles break very weird sometimes and incoming arrows overlap single letters breaking into the new line. 
- leaving Arrows numbers overlap quite quick when a few actions leave from an actor. Numbers should be aware and move accordingly
- Colors don’t always apply on first try, sometimes never
- Reordering actors/items leads to having to reorganise every action arrow which becomes quite annoying when building a new story
- Rectangular or wavey arrows should be an option to choose from
- Can I save a story and pick up where I left? I don’t really get the file saving concept of Egon yet. 

Guys this is so awesome, thanks for putting this out here. I really hope this tool will be around forever 👍🏻

**Reply by hofstef** [reply]
Hi, thanks for the encouragement and the suggestions! I'll answer your questions inline:

- opening the HTML export gives me the title and nothing on the next slide. Any known issues with that? => Seems like a bug. Please open an issue, providing some more information (Egon version, ideally an example .egn file)
- The actor/item titles break very weird sometimes and incoming arrows overlap single letters breaking into the new line. => proper hyphenation would be a dream come true (see #16) but it is not supported by the diagramming framework that we use.
- leaving Arrows numbers overlap quite quick when a few actions leave from an actor. Numbers should be aware and move accordingly => we'll work on that
- Colors don’t always apply on first try, sometimes never => seems like a bug. Again, it would be great if you could provide some detailed information, ideally in form of a new issue. Does not sound like #201 (which is fixed on main/app-latest)
- Reordering actors/items leads to having to reorganise every action arrow which becomes quite annoying when building a new story => this is a limitation of the diagramming framework
- Rectangular or wavey arrows should be an option to choose from => this is a limitation of the diagramming framework - let's hope they implement that some day but t.b.o. since the framework is primarily developed for a BPMN editor, it seems unlikely. Wavey arrows are untypical for BPMN models.
- Can I save a story and pick up where I left? I don’t really get the file saving concept of Egon yet. => Yes, see user guide [here](https://egon.io/howto#auto-save-and-creating-new-domain-stories)  and [here](https://egon.io/howto#importing-domain-stories)

**Reply by big-fudge** [reply]
Thx for the promt reply. Too bad the limitations come with the modeling framework. Will open the issues...
The color changing issue doesn't persist today so I won't bother with a bug for the moment.
